openfecwebapp/routes.py

```diff
@@ -253,12 +254,12 @@ def legal_search(query, result_type):
 def advisory_opinions(query, offset):
     result_type = 'advisory_opinions'
     results = {}
-
```

Comment:
Watch these whitspace changes, they do have meaning to separate logical code blocks.